en/releases/release-1.4.0-alpha.1.md

```diff
@@ -0,0 +1,51 @@
+---
+title: TiDB Operator 1.4.0-alpha.1 Release Notes
+---
+
+# TiDB Operator 1.4.0-alpha.1 Release Notes
+
+Release date: September 26, 2022
+
+TiDB Operator version: 1.4.0-alpha.1
+
+## Compatibility Change
+
+- Due to changes in [#4683](https://github.com/pingcap/tidb-operator/pull/4683), the volume modification feature is change to be disable by default, so need to enable this feature if you want to resize PVCs of a component.
+
+## Rolling Update Changes
+
+- Due to changes in [#4494](https://github.com/pingcap/tidb-operator/pull/4494), if you deploy TiCDC without setting the `log-file` config, upgrading TiDB Operator to v1.4.0-beta.1 causes TiCDC to rolling upgrade. 
+
+## New Feature
+
+- Support to set location labels for tidb-server automatically ([#4663](https://github.com/pingcap/tidb-operator/pull/4663), [@glorv](https://github.com/glorv))
+
+- Add new fields `spec.tikv.scalePolicy` and `spec.tiflash.scalePolicy` to scale multi TiFlash and TiKV Pods simultaneously ([#3881](https://github.com/pingcap/tidb-operator/pull/3881), [@lizhemingi](https://github.com/lizhemingi))
+
+- Add a new field `startScriptVersion` to choose start script for all components ([#4505](https://github.com/pingcap/tidb-operator/pull/4505), [@KanShiori](https://github.com/KanShiori))
+
+- Allow use shorten label names to refer to some k8s well-known labels when used in PD's location labels ([#4688](https://github.com/pingcap/tidb-operator/pull/4688), [@glorv](https://github.com/glorv))
+
+- Support PiTR for BR, point in time restore from a snapshot backup and a log backup ([#4648](https://github.com/pingcap/tidb-operator/pull/4648) [#4682](https://github.com/pingcap/tidb-operator/pull/4682) [#4694](https://github.com/pingcap/tidb-operator/pull/4694) [#4695](https://github.com/pingcap/tidb-operator/pull/4695), [@WizardXiao](https://github.com/WizardXiao))
+
+## Improvements
+
+- Optimize prometheus remoteWrite configuration for `TidbMonitor` ([#4247](https://github.com/pingcap/tidb-operator/pull/4247), [@mikechengwei](https://github.com/mikechengwei))
+
+- Add metrics port for TiFlash `Service` ([#4470](https://github.com/pingcap/tidb-operator/pull/4470), [@mikechengwei](https://github.com/mikechengwei))
+
+- Update the default value of log-file for cdc to avoid overwriting `/dev/stderr` ([#4494](https://github.com/pingcap/tidb-operator/pull/4494), [@KanShiori](https://github.com/KanShiori))
```

Comment:
```suggestion
- Update the default value of `log-file` for TiCDC to avoid overwriting `/dev/stderr` ([#4494](https://github.com/pingcap/tidb-operator/pull/4494), [@KanShiori](https://github.com/KanShiori))
```